images/macos/software-report/SoftwareReport.Android.psm1

```diff
@@ -43,124 +28,122 @@ function Get-AndroidInstalledPackages {
     return $androidInstalledPackages
 }
 
-function Build-AndroidSDKToolsTable {
-    param (
-        [Parameter(Mandatory)]
-        [object] $packageInfo
-    )
-    
-    return $packageInfo | ForEach-Object {
-        $packageInfoParts = Split-TableRowByColumns $_
-        $packageName = $packageInfoParts[0]
-        $packageDescription = $packageInfoParts[2] + ", Revision " + $packageInfoParts[1]
-        return Build-AndroidTableObject -PackageName $packageName -Description $packageDescription
+
+function Build-AndroidTable {
+    Write-Host "Build-AndroidTable"
+    $packageInfo = Get-AndroidInstalledPackages
+    Write-Host $packageInfo
+    return @(
+        @{
+            "Package" = "Android SDK Tools"
+            "Version" = Get-AndroidPackageVersions -PackageInfo $packageInfo -MatchedString "Android SDK Tools"
+        },
+        @{
+            "Package" = "Android SDK Platforms"
+            "Version" = Get-AndroidPlatformVersions -PackageInfo $packageInfo
+        },
+        @{
+            "Package" = "Android SDK Build-tools"
+            "Version" = Get-AndroidBuildToolVersions -PackageInfo $packageInfo
+        },
+        @{
+            "Package" = "Android SDK Platform-Tools"
+            "Version" = Get-AndroidPackageVersions -PackageInfo $packageInfo -MatchedString "Android SDK Platform-Tools"
+        },
+        @{
+            "Package" = "Google APIs"
+            "Version" = Get-AndroidGoogleAPIsVersions -PackageInfo $packageInfo
+        },
+        @{
+            "Package" = "Android Support Repository"
+            "Version" = Get-AndroidPackageVersions -PackageInfo $packageInfo -MatchedString "Android Support Repository"
+        },
+        @{
+            "Package" = "Google Play services"
+            "Version" = Get-AndroidPackageVersions -PackageInfo $packageInfo -MatchedString "Google Play services"
+        },
+        @{
+            "Package" = "Google Repository"
+            "Version" = Get-AndroidPackageVersions -PackageInfo $packageInfo -MatchedString "Google Repository"
+        },
+        @{
+            "Package" = "SDK Patch Applier v4"
+            "Version" = Get-AndroidPackageVersions -PackageInfo $packageInfo -MatchedString "SDK Patch Applier v4"
+        },
+        @{
+            "Package" = "CMake"
+            "Version" = Get-AndroidPackageVersions -PackageInfo $packageInfo -MatchedString "cmake"
+        },
+        @{
+            "Package" = "NDK"
+            "Version" = Get-AndroidPackageVersions -PackageInfo $packageInfo -MatchedString "ndk-bundle"
```

Comment:
You missed older NDKs. On macOS, we install 3 NKDs: https://github.com/actions/virtual-environments/blob/main/images/macos/software-report/SoftwareReport.Android.psm1#L88
I suggest to return this function back and just adapt it to return the same results like Get-AndroidPackageVersions